modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/folders/SuggestedFolders.kt

```diff
@@ -0,0 +1,37 @@
+package au.com.shiftyjelly.pocketcasts.podcasts.view.folders
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import au.com.shiftyjelly.pocketcasts.analytics.AnalyticsEvent
+import au.com.shiftyjelly.pocketcasts.analytics.AnalyticsTracker
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.extensions.contentWithoutConsumedInsets
+import au.com.shiftyjelly.pocketcasts.views.fragments.BaseDialogFragment
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class SuggestedFolders : BaseDialogFragment() {
+
+    @Inject lateinit var analyticsTracker: AnalyticsTracker
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ) = contentWithoutConsumedInsets {
+        AppThemeWithBackground(theme.activeTheme) {
+            SuggestedFoldersPage(
+                onDismiss = {
+                    dismiss()
+                },
+                onUseTheseFolders = {},
+                onCreateCustomFolders = {
```

Comment:
Would it be good to dismiss this fragment so that if you close the "Create folder" page you don't also need to close the "Suggested Folders" page?

Reply:
Sure. I had applied this for the paywall, but I think it also makes sense for this case